releasenotes/notes/ksm-resource-type-sharding-1b2e4e391ff4cca1.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
```

Comment:
This should be in the DCA release notes, not the agent ones.

Also, maybe we should mention the new option introduced to enable this.